explainaboard/utils/eval_basic.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
```

Comment:
Do we need this file?

Reply:
tested: we don't need it, let's delete it.